index.bs

```diff
@@ -563,7 +572,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
         1. Otherwise, [=list/Append=] |C| to |excludeList|.
     1. [=In parallel=], invoke the [=authenticatorMakeCredential=] operation on |authenticator| with |rpId|,
         |clientDataHash|, |options|.{{MakeCredentialOptions/rp}}, |options|.{{MakeCredentialOptions/user}},
-        |normalizedParameters|, |excludeList| and |clientExtensions| as parameters.
+        |normalizedParameters|, |excludeList|, |clientExtensions|, and |keyStorageDevice| as parameters.
```

Comment:
It is necessary to pass the bit in to authenticatorMakeCredential given that this option is part of the parameters of the authenticatorMakeCredential operation. keyStorageDevice is the name of that parameter in the CTAP spec.